search/templates/search/search.html

```diff
@@ -43,11 +43,11 @@
         var map = event.detail.map.setView(mapCenter,zoomLevel);
 
         for (var i=0; i < markerList.length; i++){
-            let g_url = markerList[i][2];
+            var location_url = markerList[i][2];
 
             marker = new L.marker([markerList[i][0], markerList[i][1]])
             .addTo(map)
-            .bindPopup(`<a target="_blank" href="${ g_url }">Open Google Maps</a>`);
+            .bindPopup(`<a target="_blank" href="${location_url}">View on Location Page</a>`);
```

Comment:
Can we change the text here to just 'View Location'?

Reply:
or maybe even display a miniature thumbnail of the apartment pic? It might help to make it clearer which apartment the user is clicking on.

Reply:
I've tried to get the image, but it needs some research at this point. Because what we can get from here is the location item, but the image is stored in the apartment.
Do you think I should start researching on it, or start doing other story and come back to it later?

Reply:
@ab7289 Actually, I did it haha
![image](https://user-images.githubusercontent.com/18141004/70344283-4fcf5380-1827-11ea-9451-49f1ce33bfa7.png)

Reply:
Nice! I think there are still some issues tho. It wasn't displaying the thumbnail when I uploaded my own apartment.

<img width="231" alt="Screen Shot 2019-12-06 at 7 43 18 PM" src="https://user-images.githubusercontent.com/32490273/70365700-b91f8880-1860-11ea-988b-499ae46578c8.png">

Reply:
<img width="412" alt="Screen Shot 2019-12-06 at 7 44 25 PM" src="https://user-images.githubusercontent.com/32490273/70365717-d48a9380-1860-11ea-8f91-2e891c3a0bd9.png">